src/styles/blocks/contacts.scss

```diff
@@ -0,0 +1,37 @@
+.contacts {
+  &__info {
+    margin: 0 0 24px;
+
+    &--last {
```

Comment:
class name shouldn't be connected with element order. use `:last-child()` instead. why only last element is `pointer`? I think you need another class for that. check in other places for this mistake